causing/examples/models.py

```diff
@@ -1,11 +1,14 @@
 # -*- coding: utf-8 -*-
 """Model Examples."""
 
-import sympy
 import numpy as np
+import sympy
 from sympy import symbols
-from causing.simulate import SimulationParams, simulate
+
 from causing.model import Model
+from causing.simulate import SimulationParams, simulate
+
+data_path = __file__.split("causing")[0]
```

Comment:
Let's hope there's only one `causing` in the path (e.g. '/home/karl/causing-dev/causing')! Something like `__file__ + '../../../'` would be safer in such a case. But the problem is not too likely, so feel free to keep it as it is.